staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -89,6 +126,38 @@ func (o *AuditLogOptions) ApplyTo(c *server.Config) error {
 		}
 	}
 	c.LegacyAuditWriter = w
-	c.AuditBackend = pluginlog.NewBackend(w)
+
+	if advancedAuditingEnabled() {
+		c.AuditBackend = appendBackend(c.AuditBackend, pluginlog.NewBackend(w))
+	}
+	return nil
+}
+
+func NewAuditWebhookOptions() *AuditWebhookOptions {
+	return &AuditWebhookOptions{Mode: pluginwebhook.ModeBatch}
```

Comment:
Changed this to default to "batch" (maybe we should rename this "async"?), since it's expected to have less impact on API server performance.